drivers/can/can_stm32h7.c

```diff
@@ -115,12 +115,12 @@ static const struct can_driver_api can_stm32h7_driver_api = {
 	.timing_min = {
 		.sjw = 0x00,
```

Comment:
This likely goes for sjw as well?

```suggestion
		.sjw = 0x01,
```

Reply:
You're right, it does.
With value 0x00 (`CAN_SJW_NO_CHANGE`) the MCAN driver retains the previous SJW value that was written in `NBTP`/`DBTP` register, which after reset is 0x03, which is not the minimal configurable value.

Changed.